Services.Test/DocumentDbKeyValueContainerTest.cs

```diff
@@ -196,7 +202,8 @@ public async Task CreateAsyncConflictTest()
                     It.IsAny<string>(),
                     It.IsAny<object>(),
                     It.IsAny<RequestOptions>(),
-                    It.IsAny<bool>()))
+                    It.IsAny<bool>(),
+                    It.IsAny<CancellationToken>()))
```

Comment:
Suggestion: I'm seeing several mock setups for .CreateDocumentAsync (and possibly others).  If you are only varying one or two of the inputs, you should consider using a mock builder function or class so you don't have to duplicate all of the It.IsAny<...>() for parameters you don't care about.